src/Raven.Server/Documents/Handlers/Debugging/DocumentDebugHandler.cs

```diff
@@ -54,5 +58,65 @@ public async Task HugeDocuments()
                 writer.WriteEndObject();
             }
         }
+        
+        [RavenAction("/databases/*/debug/documents/scan-corrupted-ids", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, IsDebugInformationEndpoint = true)]
+        public async Task ScanCorruptedIds()
+        {
+            var startEtag = GetIntValueQueryString("startEtag", required: false) ?? 0;
+            var corruptedCount = GetIntValueQueryString("corruptedCount", required: false) ?? int.MaxValue;
```

Comment:
can we have a default page size here to 1024? I don't like this being unbounded...

Reply:
Done